codeforamerica/vita-min
# Prevent 500 errors on unemployment submit without values

## Link to pivotal/JIRA issue
- No jira
## Is PM acceptance required? (delete one)
- Yes - don't merge until JIRA issue is accepted!

**Reminder**: merge main into this branch and get green tests before merging to main
## What was done?
`params#require` causes failures when a form is submitted with no values sent. Because radio values when unchecked send no values, when a user clicks nothing on the unemployment page and hits submit, the page 500s. This adds a hidden input that will always be sent and removes the 500 issue.

## How to test?
- Navigate through any flow that includes `questions/unemployment`. Click 'Continue' without clicking a radio. See no 500 error.

## Screenshots (for visual changes)
- Before
<img width="1216" alt="image" src="https://github.com/user-attachments/assets/c8512be0-6419-4db1-95d8-4eaa8a1aa5ab">

- After
<img width="1512" alt="image" src="https://github.com/user-attachments/assets/15c8b162-b4be-4fc3-8269-a74d0f4e13a8">
